packages/vite/src/node/plugins/css.ts

```diff
@@ -2170,15 +2185,16 @@ async function compileLightningCSS(
 
   const res = styleAttrRE.test(id)
     ? (await importLightningCSS()).transformStyleAttribute({
+        ...config.css?.lightningcss,
```

Comment:
I've tested that lightningCSS seems to ignore extra props. Is this safe for the future @devongovett?